.travis.yml

```diff
@@ -9,15 +7,25 @@ matrix:
   include:
     - node_js: "0.10"
       env: BROWSER=true
-env: BROWSER=false
 before_install:
   - npm install -g npm
-  - if [[ $BROWSER == true ]]; then npm install karma-phantomjs-launcher karma-firefox-launcher; fi
+  - npm install -g karma-cli
```

Comment:
Why would we install it globally instead of in `.node_modules`? The executable will still be in the PATH when running via `npm test`.